CONTRIBUTING.md

```diff
@@ -12,10 +12,10 @@ If that is the case, [open a new issue][new issue]. Make sure to include a clear
 
 ### Submitting a pull request
 Make sure that the pull request addresses an open issue in the issue tracker.  
-Remember to follow the [TEAMMATES Development Process][dev].
+Remember to follow the [TEAMMATES Development Process][dev], especially the naming conventions for pull request titles and branch names.
```

Comment:
I think it'd be better to use "in particular" rather than "especially" here, [the nuance is different](http://www.englishlessonsbrighton.co.uk/how-to-choose-between-in-particular-and-particularly/).